src/app/components/User/User.module.css

```diff
@@ -6,6 +6,7 @@
 .avatar {
   border-radius: 50%;
   margin-right: 1rem;
+  height: 3rem;
```

Comment:
If the width is calculated based on the original image size, consider to set it too to avoid layout shifting.

Reply:
true, it was only for testing reasons, until the finished layout/ design is published